docs/build.py

```diff
@@ -230,6 +233,7 @@ def _generate_docs(args, api_client):
         _generate_ansible_docs(args, api_spec, template_ctx)
     elif args.doctype == DocType.ftd_api:
         error_codes = api_client.fetch_error_codes()
+        template_ctx['error_codes'] = bool(False)
```

Comment:
Why do we need to set this variable here? Shouldn't `Error Codes` section be a part of `config.json`?

Reply:
(Changed to bool(error_codes))
Devices before 6.4 don't have error codes file, so in this case, `error_codes.md` file will not be present